kernel/kernel-api/src/main/java/io/delta/kernel/internal/actions/AddFile.java

```diff
@@ -33,21 +33,28 @@
 /**
  * Delta log action representing an `AddFile`
  */
-public class AddFile extends FileAction
-{
+public class AddFile extends FileAction {
```

Comment:
do you need the AddFile class with all the fields any more?

Reply:
Downstream consumers will be deserializing the Row (to get the path, the partition values, the stats, etc.)

We certainly need the AddFile schema .. and helpers ... but whether or not we actually need to fully deserialize 1 row into _all_ the fields remains to be seen.

I'm fine with deciding that later